helm/coco-kafka-bridge/app-configs/kafka-bridge_eks_delivery_dev.yaml

```diff
@@ -27,11 +27,4 @@ bridges:
     type: "proxy"
     replicas: 2
   # MSK -> Kafka
```

Comment:
We can also get rid of these comments since this was our last bridge for the environment.